### PR TITLE
Bootstrap fix

### PR DESCRIPTION
**Pull request by mabmalia**
We need to include this script line in the index.html file, otherwise bootstrap functionalities that requires javaScript won't work.

Example: if you reduce the size of the page, the nav bar is replaced by a button. If you click in the button without this script line, it will not show the menu options.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
+#Wed Nov 27 15:07:39 CET 2019
+=======
```
Comment (elmenqui) [inline gradle/wrapper/gradle-wrapper.properties]: These are GitHub added text and will break the code

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
 #Wed Nov 27 19:09:26 CET 2019
+>>>>>>> master
```
Comment (elmenqui) [inline gradle/wrapper/gradle-wrapper.properties]: Same as the above comment

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (mabmalia) [reply gradle/wrapper/gradle-wrapper.properties]: Did you fetch this code?

**Review by elmenqui — APPROVED** [review]
LGTM

**Review by elmenqui — APPROVED** [review]
LGTM

**Review by MeriM-dev — APPROVED** [review]
LGTM

**Pull request merged**